### ISSUE TITLE
BUG: Error when selecting the "Tags" field in the settings

### ISSUE THREAD

**Issue by BorisNA** [mortii/anki-morphs#98]
**Describe the bug**
Exception when clicking on the "tags" field in Settings.

**To Reproduce**
1. Open Settings
2. Click on the "rags" field 
3. Get an exception

Caught exception:
```
Traceback (most recent call last):
  File "C:\_MyPrograms\Anki2\Data\addons21\472573498\settings_dialog.py", line 722, in _tags_cell_clicked
    note_type = mw.col.models.get(NotetypeId(int(current_model_id)))
  File "C:\_MyPrograms\Anki2\Data\addons21\472573498\tag_selection_dialog.py", line 71, in set_selected_tags_and_row
    excluded: list[str] = tag_object["exclude"]
TypeError: list indices must be integers or slices, not str
```

![image](https://github.com/mortii/anki-morphs/assets/7961527/802e5c06-8445-4179-a35f-442717018b83)

**Desktop (please complete the following information):**
Anki 2.1.66 (70506aeb) Python 3.9.15 Qt 6.5.2 PyQt 6.5.2

Platform: Windows-10-10.0.19045
Flags: frz=True ao=True sv=3
Add-ons, last update check: 2023-12-13 18:58:04
Add-ons possibly involved: ⁨ankimorphs-alpha⁩

**Additional context**
My meta.json with the config is attached
[meta.json](https://github.com/mortii/anki-morphs/files/13662913/meta.json)

**Comment by mortii** [comment]
@BorisNA Thanks for reporting it!

Your settings are outdated, you can fix it in three ways:
1. delete meta.json
2. Tools -> AnkiMorphs -> Settings -> Restore all default settings
3. Tools -> Add-ons -> Select ankimorphs-alpha -> Config -> Restore Defaults

After that you have to redo the settings

**Comment by BorisNA** [comment]
Oh, my bad, should have thought about it.
Anyway, after I edited meta.json for the "tags" key, I got a big message that I should update settings ... it looks like it was blocked by the exceptions mentioned above.

P.S. Sorry for hijacking the thread, I'm a littel bit  lazy to open another issue. It looks like there should be at config.py:13 "from . import ankimorphs_globals" and not "from ankimorphs import ankimorphs_globals"
At least it throws another exception for me.

**Comment by mortii** [comment]
> P.S. Sorry for hijacking the thread, I'm a littel bit lazy to open another issue. It looks like there should be "from . import ankimorphs_globals" and not "from ankimorphs import ankimorphs_globals" at config.py:13 . At least it throws another exception for me.

@BorisNA holy, I have no idea how that happened.... Thanks for letting me know! I pushed a hotfix, does everything work for you now?

**Comment by BorisNA** [comment]
Yes, it works perfectly. Thanks alot.
